VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -1319,6 +1319,415 @@ Function Install-EsxiCertificate {
     }
 }
 
+Function Set-SddcCertificateAuthority {
```

Comment:
Perhaps this should be `Set-SddcMSCertificateAuthority`; however, having the option for a `-type` switch to specify `Microsoft` or `OpenSSL` would be more beneficial. If so, the metadata and examples would need to be updated - and obviously the cmdlet logic.

Reply:
openSSL support is not implemented yet, will open an issue on this